src/karmen_frontend/src/components/printers/webcam-stream.js

```diff
@@ -12,7 +12,7 @@ const WebcamModal = ({ classNames, source, url, allowFullscreen }) => {
   return (
     <>
       <img
-        className={classNames.join(" ")}
+        className={"pointer" + classNames.join(" ")}
```

Comment:
this works? for example:
```
classNames = ["one", "two"];
console.log("pointer" + classNames.join(" "));
> "pointerone two"
```
Asi by mělo být `classNames.concat("pointer").join(" ")`

Reply:
333616b